country-map.js

```diff
@@ -307,6 +231,10 @@ module.exports = {
         "abbreviation": "CZ",
         "currency": "USD"
     },
+    "Denmark": {
+        "abbreviation": "DK",
+        "currency": "USD"

```

Comment:
Good catch! Looks like changes for countries starting alphabetically before "Greece" didn't make it into the last commit because of a copy paste error.